src/cmd/serve.rs

```diff
@@ -229,7 +229,9 @@ pub fn serve(interface: &str, port: &str, output_dir: &str, base_url: &str, conf
         watchers.push("sass");
     }
 
-    println!("Listening for changes in {}/{{{}}}", pwd.display(), watchers.join(", "));
+    let mut pwd_directory = pwd.clone();
+    pwd_directory.push(""); // add OS-specific trailing slash
```

Comment:
How difficult would it be to use something like https://doc.rust-lang.org/std/path/constant.MAIN_SEPARATOR.html? Rather than cloning the data and mutating it? Not sure how crazy complicated that'd make the print formatting though?

Reply:
+1 for this approach

Reply:
Thanks for the hint! It's pretty straightforward, and doesn't look any more complicated. Added a corrected commit to the PR.